src/app/pages/categories/management/pages-categories-management.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="container">
+
+  <div class="loading-container" *ngIf="transactionCategoriesData.isLoading">
+    <mat-spinner [diameter]="30"></mat-spinner>
+  </div>
+
+  <div class="message-container" *ngIf="transactionCategoriesData.hasError">
+    <app-system-message [type]="notificationTypes.Error"
+                        text="Something went wrong. We were not able to get your wallets"
+                        description="Please try refreshing your browser"></app-system-message>
+  </div>
+
+  <div class="categories__container" *ngIf="transactionCategoriesData.data">
+
+    <h2 class="categories__title">
```

Comment:
I think this is obsolete. We already know the page we are on based on active element in header navigation.

Reply:
Was not done yet

Reply:
Aaaaand still not done, hello there.